src/manifest/pfm.rs

```diff
@@ -967,4 +966,65 @@ mod test {
         assert_eq!(imgs[1].region(1), Some(Region::new(0x28000, 0x1000)));
         assert!(imgs[1].region(2).is_none());
     }
+
+    #[test]
+    fn baked_pfm1() {
+        let sha = ring::sha256::Builder::new();
+        let (mut rsa, mut signer) = test_rsa();
+        let mut arena = vec![0; 1024];
+        let arena = BumpArena::new(&mut arena);
+
+        let bytes = Ram(testdata::PFM_RSA1);
```

Comment:
Since this is a generated binary resource, maybe some comments which describe what one should expect?

Doesn't necessarily have to be here - perhaps in testdata/mod.rs next to the declaration of PFM_RSA1.